forms/form.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+
+
+class FormForms(forms.Form):
```

Comment:
1) Please this file at `forms/dtos.py`
2) `FormForms` -> `FormCreateDto`*

Please google about DTOs or ask me in Telegram

Reply:
Did it with love

Reply:
> Please this file at forms/dtos.py

do not finished

Reply:
This renaming is cooler than cool eggs